### PR TITLE
DCS-1350 Fixing display of prisoner image.

### PR DESCRIPTION
**Pull request by andrewrlee**
Also showing PNC and Prison number of arrival

### CODE REVIEW

File: server/views/pages/bookedtoday/arrivals/searchforexisting/multipleExistingRecordsFound.njk [hunk 1]
```diff
@@ -56,7 +56,7 @@
                                                 <label class="govuk-label govuk-radios__label govuk-!-padding-right-0 govuk-!-padding-top-0" for="record-{{loop.index}}">
                                                     <div class="prisoner-record-detail flex">
                                                         {{ prisonerSummaryDetail(match, "matching-record-"+loop.index) }}
-                                                        <img src="{{ '/prisoner/' + match.prisonNumber + '/image' if match.prisonNumber else '/assets/images/placeholder-image.png' }}" 
+                                                        <img src="{{ '/prisoners/' + match.prisonNumber + '/image' if match.prisonNumber else '/assets/images/placeholder-image.png' }}"
```
Comment (andrewrlee) [inline server/views/pages/bookedtoday/arrivals/searchforexisting/multipleExistingRecordsFound.njk:59]: this was preventing images being displayed

File: server/routes/bookedtoday/arrivals/searchforexisting/multipleExistingRecordsFoundController.ts [hunk 2]
```diff
@@ -10,16 +10,16 @@ export default class MultipleExistingRecordsFoundController {
...
       const potentialMatches = await this.expectedArrivalsService.getMatchingRecords(searchData)
 
       res.render('pages/bookedtoday/arrivals/searchforexisting/multipleExistingRecordsFound.njk', {
-        arrival,
+        arrival: {
+          firstName: searchData.firstName,
+          lastName: searchData.lastName,
+          dateOfBirth: searchData.dateOfBirth,
+          prisonNumber: searchData.prisonNumber,
+          pncNumber: searchData.pncNumber,
```
Comment (andrewrlee) [inline server/routes/bookedtoday/arrivals/searchforexisting/multipleExistingRecordsFoundController.ts:21]: these were missed on the arrival, wasn't very obvious that they needed to be displayed when populated

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Review by swilson-mojhub — APPROVED** [review]
Looks good

**Pull request merged**